java/spacewalk-java.changes.hemantyb.fix-generic-parameter

```diff
@@ -0,0 +1,2 @@
+- Fix: fix the generics for set, iterator, DataResult and others,
```

Comment:
This changelog has a different purpose from a git commit, we use it to generate user-facing changelogs that are shipped with the release. As such, this changelog should reflect feature changes and be aimed at an end user, not developers. As is, I would recommend omitting this changelog altogether since there's no user-facing changes.